tmt/steps/prepare/__init__.py

```diff
@@ -200,6 +205,7 @@ def go(self) -> None:
             # operations inside prepare plugins on the guest use the
             # prepare step config rather than provision step config.
             guest_copy = copy.copy(guest)
+            guest_copy._logger = guest._logger.clone().apply_verbosity_options(**self._options)
```

Comment:
Shall we use `inject_logger` here?

Reply:
Correct, will fix.

Reply:
Fixed in 39891de6dbf9fb4e01471bda84147df2156d9eea